README.md

```diff
@@ -44,6 +44,18 @@ Stop geOrchestra with
 docker-compose down
 ```
 
+## SSL/TLS
+
+This repo comes with a self signe cert which is _not_ valid. To test with a valid cert you can
+install [mkcert](https://github.com/FiloSottile/mkcert) and do the following:
+
+* `mkcert -install`. Only do this once ! It'll install a fake root cert in system store (and some
+  other see mkcert doc)
```

Comment:
some others ?

Reply:
Firefox  (and maybe Chrome, but not and expert). But I'd like to be as concise here as possible and if people are interested, they'll check the doc.

Reply:
I meant "some others", not "some other", for the typo ;-)

Reply:
:sweat_smile: